src/main/java/edu/harvard/iq/dataverse/license/LicenseListingPage.java

```diff
@@ -55,4 +58,26 @@ public String init() {
         return StringUtils.EMPTY;
     }
 
+    public void saveLicenseActiveStatus(LicenseDto licenseDto) {
+
+        if (licenses.stream().noneMatch(LicenseDto::isActive)) {
+            licenseDto.setActive(true);
+            displayNoLicensesActiveWarningMessage();
+            return;
+        }
```

Comment:
Do not rely on dto's when checking such things. They can be out of sync. Get that info from database.